build.sbt

```diff
@@ -201,8 +200,23 @@ lazy val plugin = project
     )
   )
 
-lazy val tests = crossProject(JSPlatform, JVMPlatform, NativePlatform)
+lazy val munitScalacheck = crossProject(JSPlatform, JVMPlatform, NativePlatform)
+  .in(file("munit-scalacheck"))
+  .settings(
+    sharedSettings,
+    crossScalaVersions := List(scala213, scala212, scala211),
```

Comment:
no dotty support since there are no binaries published and I couldn't make it work with `withDottyCompat` :(

Reply:
ah, nevermind. The issue I had with `withDottyCompat` seems to be fixed in Dotty 0.23 🎉